### DISCUSSION TITLE
关于 file to file 的 checkpoint 记录现象

### DISCUSSION THREAD

**Discussion by MamyAKKK** [alibaba/ilogtail#1849] [Help] [answered]
在docker集群中部署了ilogtail去采集业务系统日志目录（已收集到存储中的目录）下指定业务日志文件，然后将其输出到指定目录的指定文件下（业务需求）。

现状是业务源日志文件不停生成，ilogtail 采集也没问题，但是checkpoint很久不更新，至少1个小时未更新过，就容器启动时创建了checkpoint元数据。
多次删除创建ilogtail容器后，诡异的地方来了，有时候是会重头开始采集，有时候又从当前时间开始采集。暂未发现规律，所以想知道这个checkpoint到底该如何配置？能否设置频率或者其他参数？

PS : 
1 . ilogtail version 2.0.7 From Github.
2 . iogtail image os : CentOS7.9.
3 . ilogtail 所有的配置都是由 NFS 服务支持。

Thanks~

**Comment by messixukejia** [comment, marked as answer]
可以看下这篇文章https://developer.aliyun.com/article/901257

**Reply by MamyAKKK** [reply]
非常感谢，我按照Github的DOC找了半天，没有任何有用的信息，我按照你的提示，找了新的checkpoint项，顺着这个信息，我在SLS官方帖中找到更多关于它的其他核心配置，原来Github上只是非常少的一小部分。再次感谢您~

**Reply by messixukejia** [reply]
如果方便，欢迎登记下https://github.com/alibaba/ilogtail/issues/693

也可以帮忙把github上文档完善下，让更多人受益。也可以把上述文档提交到https://github.com/alibaba/ilogtail/blob/main/docs/cn/awesome-ilogtail/ilogtail.md
